pipelines/pom.xml

```diff
@@ -105,14 +133,6 @@
     <dependency>
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
-      <version>${logback.version}</version>
-      <scope>runtime</scope>
```

Comment:
nit: Any particular reason for dropping `runtime` scope? One benefit of this scope is that it makes sure we do not directly use `logback` APIs in our code (and instead do everything through `slf4j`).

Reply:
I think this makes sense, I will add the scope as `runtime` for the changes that I did.

Reply:
Added the scope as `runtime` for the `dependencies` in the `dependencyManagement` section, which gets inherited for the `dependencies`